.github/actions/setup-sql-server/Initialize-SqlServer.ps1

```diff
@@ -1,18 +1,20 @@
 if ($Env:RUNNER_OS -eq 'Windows')
 {
-    choco install sql-server-express --no-progress
+    choco install sql-server-2022 --no-progress
 }
 else
 {
+    $sqlServerName = 'sql2022'
+    $sqlServerVersion = '2022-latest'
     $dockerRunSwitches = @(
-        '--name', 'sql2019'
+        '--name', $sqlServerName
```

Comment:
Set the container name here, value 'uitt-sqlserver'. Also see my comment below (`Invoke-SolutionTests.ps1:L25`).